dashboard/app/jobs.go

```diff
@@ -255,7 +268,10 @@ func createBisectJob(c context.Context, managers map[string]bool, reproLevel das
 		if crash == nil {
 			continue
 		}
-		return createBisectJobForBug(c, bug, crash, keys[bi], crashKey)
+		if jobType == JobBisectFix && timeSince(c, bug.LastTime) < 24*30 {
```

Comment:
We again may all 300 jobs that will be filtered out by this condition and then fix bisection will stop. We sort by FirstTime (newer first), so these have recent LastTime too.
Perhaps we need to order fix bisections by LastTime instead so that we get the oldest LastTime first. This would ensure that if there are any old LastTime's, we will fetch them among these 300 bugs.

Reply:
Thanks, I've made a change such that for fix bisection, bugs are filtered by LastTime and sorted by LastTime(oldest first).